### PR TITLE
Test with CALLDATACOPY where call data offset > call data length, length to copy > 0

### PR DESCRIPTION
**Pull request by jwasinger**
The resulting behavior is that `length` amount of memory is zeroed out.

In ethereumjs-vm I was able to completely clear memory in this case without causing test failures: https://github.com/ethereumjs/ethereumjs-vm/pull/174/commits/5366bc4254b72523f8347bf68d0cd494212173ce

### CODE REVIEW

File: src/GeneralStateTestsFiller/stCodeCopyTest/codeCopyOffsetFiller.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                    }
+                }
+            }
+        ],
+        "pre" : {
+						"eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" : {
+                "balance" : "1000000000000000000",
+								"//codeDescription": "/* fill memory up */ /* copy call data to memory with offset > call data length */  /* only part of memory should be zeroed out */",
+								"code": "{ (MSTORE 0x00 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff) (CALLDATACOPY 0x100 0xffff  0x10) (SSTORE 0x00 (MLOAD 0x00)) }",
+                "nonce" : "1",
```
Comment (pirapira) [inline src/GeneralStateTestsFiller/stCodeCopyTest/codeCopyOffsetFiller.json]: What happened to the indentation around here?

File: src/GeneralStateTestsFiller/stMemoryTest/callDataCopyOffsetFiller.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            }
+        ],
+        "pre" : {
+						"eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" : {
+                "balance" : "1000000000000000000",
+								"//codeDescription": "/* fill memory up */ /* copy call data to memory with offset > call data length */  /* only part of memory should be zeroed out */",
+								"code": "{ (MSTORE 0x00 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff) (CALLDATACOPY 0x00 0xffff  0x10) (SSTORE 0x00 (MLOAD 0x00)) }",
+                "nonce" : "1",
+								"storage": {
+								}
```
Comment (pirapira) [inline src/GeneralStateTestsFiller/stMemoryTest/callDataCopyOffsetFiller.json]: And around here, too?

Sometimes I read test fillers on GitHub, and currently, I need to scroll to the right.

**Comment by jwasinger** [comment]
Okay @pirapira I think I have corrected the issues now, as well as adding blockchain test and filler.

File: src/GeneralStateTestsFiller/stCodeCopyTest/codeCopyOffsetFiller.json (hunk 1 above)
Comment (jwasinger) [reply src/GeneralStateTestsFiller/stCodeCopyTest/codeCopyOffsetFiller.json]: Okay @pirapira I think I have corrected the issue now as well as removing unwanted file, adding blockchain test and filler.

**Pull request merged**